pkg/cmd/clients/factory.go

```diff
@@ -416,6 +416,7 @@ func (f *factory) ResetSecretsLocation() {
 
 // CreateSystemVaultClient gets the system vault client for managing the secrets
 func (f *factory) CreateSystemVaultClient(namespace string) (vault.Client, error) {
+	log.Logger().Debugf("Creating system vault for namespace %s", namespace)
```

Comment:
one thing I keep wondering about is capitalization and punctuation of our logging messages. We seem to be all over the place atm. IMO it would be nice to establish some guidelines. 

(personally, I favour not capitalization for logging and errors)

Reply:
I agree, some logging standards would be a good idea. I'm happy to remove the capitalisation if that's what we're going with.

Reply:
> I agree, some logging standards would be a good idea. I'm happy to remove the capitalisation if that's what we're going with.

I guess for now it does not matter. Hopefully, we have a plan forward after the CLI overhaul discussion.